uktrade/data-hub-frontend
# feature/CLS2-428-investment-projects-tasks-view

## Description of change

- Add a new sub page in the investment project details page to display tasks assigned to the investment project.
- Added a link in the investment project details navigation menu, that is feature flagged
- Added conditional checks in the `ProjectLayout` component to handle the scenario of an investment project loading

## Test instructions

Create some tasks for an investment project, or use a project that already has some: http://localhost:3000/investments/projects/abf2c1f4-0c43-4739-86e2-f9d12556a497/tasks. The pagination, Add task and task links should all be functional.
To view this link, your user needs to have the 'adviser-tasks' feature flag assigned in the api

## Screenshots

### Before

NA

### After

![image](https://github.com/uktrade/data-hub-frontend/assets/102232401/af5dad9a-e662-41c5-b56c-fb0ce07f368d)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
